### PR TITLE
Update welsh individual translations

### PR DESCRIPTION
**Pull request by ajmaddaford**
### What is the context of this PR?
Update the Census Welsh individual translations with the latest download from Crowdin.

### How to review 
Check that the translations look correct.

### CODE REVIEW

File: app/translations/cy/census_individual_gb_wls_cy.po [hunk 1]
```diff
@@ -210,13 +214,13 @@ msgid "We are measuring the number of people who have served in the UK Armed For
 msgstr "Rydym ni’n mesur faint o bobl sydd wedi gwasanaethu yn Lluoedd Arfog y Deyrnas Unedig ac sydd bellach wedi gadael. Mae angen i’r llywodraeth a chynghorau gael y wybodaeth hon er mwyn cyflawni eu hymrwymiadau o dan Gyfamod y Lluoedd Arfog. Addewid gan y genedl yw hwn, sy'n sicrhau nad yw’r rhai sy'n gwasanaethu neu sydd wedi gwasanaethu yn y lluoedd arfog, na'u teuluoedd, o dan anfantais."
 
 msgid "Current serving members should only select “No”"
-msgstr "Aelodau sy'n gwasanaethu ar hyn o bryd, dewiswch \"Nac ydw\" yn unig"
+msgstr "Aelodau sy'n gwasanaethu ar hyn o bryd, dewiswch \"Nac ydw\"/\"Nac ydy\" yn unig"
```
Comment (MebinAbraham) [inline app/translations/cy/census_individual_gb_wls_cy.po:217]: It's Welsh so not a clue but this looks dodgy?. Two lots of No?
If it is what they have done I guess that what we accept?

File: app/translations/cy/census_individual_gb_wls_cy.po [hunk 2]
```diff
@@ -1579,12 +1583,12 @@ msgstr "Cymraeg neu Saesneg"
 
 #. answer-id: language-answer
 msgctxt "Answer for: What is <em>{person_name_possessive}</em> main language?"
-msgid "Other"
-msgstr "Arall"
+msgid "Other, including British Sign Language"
+msgstr "Arall, gan gynnwys laith Arwyddion Prydain"
```
Comment (berroar) [inline app/translations/cy/census_individual_gb_wls_cy.po]: There is a typo here- the word 'laith' should be 'iaith/Iaith'.

**Review by rmccar — APPROVED** [review]
Im not exactly fluent in Welsh but looks ok to me

File: app/translations/cy/census_individual_gb_wls_cy.po (hunk 1 above)
Comment (ajmaddaford) [reply app/translations/cy/census_individual_gb_wls_cy.po:217]: The answer option is "No" in English for both proxy and non-proxy, for Welsh the answer option is "Nac Ydw" for non-proxy and "Nac Ydy" for proxy. It looks like this has been done as we don't have translation context for guidance contents. I'll check if we have a card for that and raise one if we don't.

File: app/translations/cy/census_individual_gb_wls_cy.po (hunk 2 above)
Comment (ajmaddaford) [reply app/translations/cy/census_individual_gb_wls_cy.po]: Good spot! Fixed.

**Review by berroar — APPROVED** [review]

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**